test/ffmpeg-qsv/encode/avc.py

```diff
@@ -240,3 +240,49 @@ def init(self, tspec, case, rcmode, bitrate, maxrate, qp, quality, profile):
   def test(self, case, rcmode, bitrate, maxrate, qp, quality, profile):
     self.init(spec, case, rcmode, bitrate, maxrate, qp, quality, profile)
     self.encode()
+
+class intr_ref(AVCEncoderTest):
+  def init(self, tspec, case, gop, bframes, bitrate, qp, maxrate, profile, rcmode, ref_type, ref_size, ref_dist):
+    vars(self).update(tspec[case].copy())
+    vars(self).update(
+      rcmode     = rcmode,
+      bframes    = bframes,
+      bitrate    = bitrate,
+      case       = case,
+      maxrate    = maxrate,
+      minrate    = bitrate,
+      profile    = profile,
+      qp         = qp,
+      gop        = gop,
+      ref_type   = ref_type,
+      ref_size   = ref_size,
+      ref_dist   = ref_dist,
```

Comment:
With other suggestions, do:
```python
   intref = dict(type = reftype, size = refsize, dist = refdist)
```

Reply:
Thank you for your patience and detailed reply. (●'◡'●)